internal/pkg/cli/flag.go

```diff
@@ -278,6 +279,7 @@ Must be either "MySQL" or "PostgreSQL".`
 	taskRoleFlagDescription      = "Optional. The ARN of the role for the task to use."
 	executionRoleFlagDescription = "Optional. The ARN of the role that grants the container agent permission to make AWS API calls."
 	envVarsFlagDescription       = "Optional. Environment variables specified by key=value separated by commas."
+	envFileFlagDescription       = "Optional. A path to an environment variable (.env) file with each line being of the form of VARIABLE=VALUE. Values specified with --env-vars take precedence over --env-file."
```

Comment:
nit: can we split the description into multiple lines? we typically try to maintain a single line printed to the terminal to be <= 80 chars as that's the default terminal width.

```suggestion
	envFileFlagDescription       = `Optional. A path to an environment variable (.env) file. 
Each line should be of the form of VARIABLE=VALUE. 
Values specified with --env-vars take precedence over --env-file.`
```